src/components/ZeroState/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FDS from 'lib/dictionary/js/styleConstants';
+
+const ZeroState = ({ Icon, label, height, ...props }) => (
```

Comment:
Since this component is primarily rendered as a replacement for other content, could that logic live within ZeroState?

something like the below where children is the component normally rendered when results are available
```
return props.resultsCount ? children : <article
    className="alignChild--center--center pointer-events-none"
    style={{ minHeight: height }}
  >
    <section {...props}>
      {Icon && (
        <div className="alignChild--center--center" role="img">
          <Icon size="xl" color={FDS.FONT_COLOR_HINT} />
        </div>
      )}
      <p className="color--hint padding--top--half align--center">{label}</p>
    </section>
  </article>
```

Reply:
Thinking of it more, this may be a bit too restrictive to the usage of ZeroState and probably would be better suited for some cbi-site helper functional component